src/rapids_metadata/__init__.py

```diff
@@ -155,6 +155,139 @@
+                "libcugraphops": RAPIDSPackage(),
+                "pylibcugraphops": RAPIDSPackage(),
+            }
+        ),
+        "cuml": RAPIDSRepository(
+            packages={
+                "cuml": RAPIDSPackage(),
+                "libcuml": RAPIDSPackage(),
+                "libcuml-tests": RAPIDSPackage(),
+            }
+        ),
+        "cumlprims_mg": RAPIDSRepository(
+            packages={
+                "libcumlprims": RAPIDSPackage(),
+            }
+        ),
+        "cuproj": RAPIDSRepository(
+            packages={
+                "cuproj": RAPIDSPackage(),
+            }
+        ),
+        "cuspatial": RAPIDSRepository(
+            packages={
+                "cuspatial": RAPIDSPackage(),
+                "libcuspatial": RAPIDSPackage(),
+                "libcuspatial-tests": RAPIDSPackage(),
+            }
+        ),
+        "cuvs": RAPIDSRepository(
+            packages={
+                "cuvs": RAPIDSPackage(),
+                "libcuvs": RAPIDSPackage(),
+            }
+        ),
+        "cuxfilter": RAPIDSRepository(
+            packages={
+                "cuxfilter": RAPIDSPackage(),
+            }
+        ),
+        "dask-cuda": RAPIDSRepository(
+            packages={
+                "dask-cuda": RAPIDSPackage(has_cuda_suffix=False),
+            }
+        ),
+        "kvikio": RAPIDSRepository(
+            packages={
+                "libkvikio": RAPIDSPackage(),
+                "kvikio": RAPIDSPackage(),
+            }
+        ),
+        "ptxcompiler": RAPIDSRepository(
+            packages={
+                "ptxcompiler": RAPIDSPackage(publishes_prereleases=False),
+            }
+        ),
+        "pynvjitlink": RAPIDSRepository(
+            packages={
+                "pynvjitlink": RAPIDSPackage(),
+            }
+        ),
+        "raft": RAPIDSRepository(
+            packages={
+                "libraft": RAPIDSPackage(),
+                "libraft-headers": RAPIDSPackage(),
+                "pylibraft": RAPIDSPackage(),
+                "raft-dask": RAPIDSPackage(),
+            }
+        ),
+        "rapids-dask-dependency": RAPIDSRepository(
+            packages={
+                "rapids-dask-dependency": RAPIDSPackage(has_cuda_suffix=False),
+            }
+        ),
+        "rmm": RAPIDSRepository(
+            packages={
+                "librmm": RAPIDSPackage(),
+                "rmm": RAPIDSPackage(),
+            }
+        ),
+        "ucxx": RAPIDSRepository(
+            packages={
+                "distributed-ucxx": RAPIDSPackage(),
+                "libucxx": RAPIDSPackage(),
+                "ucxx": RAPIDSPackage(),
+            }
+        ),
+        "ucx-py": RAPIDSRepository(
+            packages={
+                "ucx-py": RAPIDSPackage(),
+            }
+        ),
+        "wholegraph": RAPIDSRepository(
+            packages={
+                "pylibwholegraph": RAPIDSPackage(),
+            }
+        ),
+    }
+)
```

Comment:
Each version should be a diff against the previous version.

```suggestion
all_metadata.versions["24.10"] = deepcopy(all_metadata.versions["24.08"])
all_metadata.versions["24.10"].repositories["cudf"].packages["pylibcudf"] = RAPIDSPackage()
all_metadata.versions["24.10"].repositories["cuvs"] = RAPIDSRepository(
    packages={
        "cuvs": RAPIDSPackage(),
        "libcuvs": RAPIDSPackage(),
    }
)
```